lib/usdUfe/ufe/Utils.h

```diff
@@ -477,6 +479,12 @@ void removeSessionLeftOvers(
     UsdUndoableItem*              undoableItem,
     bool                          extraEdits = true);
 
+//! Return the USD prims predicate for the given UFE child filter.
+//! Note: an empty filter or unknown filter will filter out everything.
+//!       Yes, that means no-filter actually means filter everything out.
+USDUFE_PUBLIC
+PXR_NS::Usd_PrimFlagsPredicate getUsdPredicate(const Ufe::Hierarchy::ChildFilter& childFilter);
```

Comment:
Looking at the comment, as long as UsdHierarchyHandler::childFilter() is accessible and will always include-all I think this should be okay. May be there's a way to guarantee include-all with a function without having to repeat the emplace_back() of filters in various code locations (if it comes to that).